OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -510,39 +515,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                only the spine and the back edge of the boards.
+                              </decoNote>
+                              <decoNote xml:id="b2" type="Boards">
+                                The boards are flush with the bookblock and have internal shallow bevels.
+                              </decoNote>
+                              <decoNote xml:id="b3" type="Cover" color="reddish-brown">
+                                The former blind tooled cover is preserved, despite deteriorated, under the present one. The decoration is partially
+                                legible on the edges, where a pattern of double circles is found.
+                                The present leather is blind-tooled as well but covers only the spine and partially the boards.
+                                On its surface bands of two <term key="singleStraightLine">single straight lines</term> intersect
+                                forming a column of juxtaposed panels. Within each panel, the lines cross each other forming a crux decussata and at the intersections of the lines,
+                                <term key="doubleCircle">double circles</term> are stamped. The back features only straight single lines outlining its perimeter.
+                                The edges of the thick leather cover are nailed to the boards and decorated with double circles.
+                              </decoNote>
+                              <decoNote xml:id="b4" type="Endbands" color="brown">
+                                Brown <term key="slitBraid">slit-braid</term> endbands are present.
+                              </decoNote>
+                              <decoNote xml:id="b5" type="SewingStations">4</decoNote>
+                              <decoNote xml:id="b6">
+                                Chainstitch sewing on four sewing stations (two pairs); periodic fold pattern. Two S-plied threads are used.
+                                The sewing starts from the first quire where, for each pair of sewing stations, there is one length of thread.
+                                In the rest of the quires the lenghts are two. The ends of the thread are knotted in the last quire.
+                              </decoNote>
+                              <decoNote xml:id="b7" type="Spine">The stitches of four endband chains are sewn into the leather of the spine, two being close to the head and two to the tail.</decoNote>
+                              <decoNote xml:id="b8" type="Other">
+                                A blue <term key="leafTabMark">leaf tab marker</term> for navigating the text is inserted in the upper corner of
+                                <locus target="#26"/>, incipit page of the reading for Tuesday, <locus target="#54"/>, <locus target="#60"/>, and <locus target="#65"/>.
+                                <!--Does someone want to add details about the other incipits?-->
```

Comment:
Here we have:
`<locus target="#54"/> (incipit page of the reading for Friday), <locus target="#60"/> (incipit page of the reading for Saturday), and <locus target="#65"/> (incipit page of the reading for Sunday)`

Reply:
Thank you!